source/Halibut.Tests/WhenTheTcpConnectionIsKilledWhileWaitingForTheResponse.cs

```diff
@@ -12,33 +11,24 @@ namespace Halibut.Tests
     public class WhenTheTcpConnectionIsKilledWhileWaitingForTheResponse
     {
         [Test]
-        public async Task ToPolling_AResponseShouldBeQuicklyReturned()
+        public async Task ToPolling_AResponseShouldBeQuicklyReturned2()
         {
-            var services = new DelegateServiceFactory();
             DoSomeActionService doSomeActionService = new DoSomeActionService();
-            services.Register<IDoSomeActionService>(() => doSomeActionService);
-            using (var octopus = new HalibutRuntime(Certificates.Octopus))
+            using (var clientAndService = ClientServiceBuilder.Polling()
+                       .WithService<IDoSomeActionService>(doSomeActionService)
+                       .WithPortForwarding(port => new PortForwarder(new Uri("https://localhost:" + port), TimeSpan.Zero))
```

Comment:
This line looks to be mostly default behaviour - can we put it in the builder (i.e. `.WithPortForwarding()`), along with an overload to take a factory function?

Reply:
Yes